src/testRunner/unittests/tsserver/partialSemanticServer.ts

```diff
@@ -172,5 +172,36 @@ function fooB() { }`
             openFilesForSession([file2], session);
             checkProjectActualFiles(project, [libFile.path, file2.path, file3.path]);
         });
+
+        it("should not create autoImportProvider or handle package jsons", () => {

```

Comment:
```suggestion
        it("should not create autoImportProvider or handle package jsons in PartialSemantic mode", () => {
```

Reply:
Its already in describe block for with `unittests:: tsserver:: Semantic operations on PartialSemantic server`

Reply:
Ah, missed what file this was.